src/data/courseDependenciesData.js

```diff
@@ -3,6 +3,73 @@ import {find} from 'lodash'
 const courseDependencyData = (courseSlug) =>
   find(
     [
+      {
+        id: '425791',
+        type: 'playlist',
+        guid: '93bd',
+        slug: 'declarative-uis-without-css-with-elm-ui-93bd',
+        dependencies: {
+          elm: '>=0.19.1',
+        },
+        freshness: {
+          status: `fresh`,
+          title: `This is a Fresh Course`,
+          text: `This course is new and up-to-date. It will provide you with a different paradigm to viewing design, layout, and CSS. .`,
```

Comment:
removing double dot `.` typo

```suggestion
          text: `This course is new and up-to-date. It will provide you with a different paradigm to viewing design, layout, and CSS.`,
```